specification/ai/Azure.AI.Unified/datasets/routes.tsp

```diff
@@ -0,0 +1,132 @@
+import "@typespec/rest";
+import "@azure-tools/typespec-autorest";
+import "@typespec/versioning";
+import "@azure-tools/typespec-azure-core";
+import "../common/models.tsp";
+import "./models.tsp";
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using TypeSpec.Versioning;
+using Azure.Core;
+using Azure.Core.Traits;
+using Azure.Core.Foundations;
+
+namespace Azure.AI.Unified;
+
+interface Datasets {
+  #suppress "@azure-tools/typespec-azure-core/operation-missing-api-version"
```

Comment:
Why are we suppressing things that is telling us that there are requirements for any Azure based service API not being fulfilled?